File: NilDevStudio.Domain/Speaker.cs
using System.Collections.Generic;

namespace NilDevStudio.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Curriculum { get; set; }
        public string ImageURL { get; set; }
        public string Telephone { get; set; }
        public string Email { get; set; }
        public List<SocialNetwork> SocialNetworks { get; set; }
        public List<EventSpeaker> EventSpeaker { get; set; }
    }
}

File: NilDevStudio-App/src/app/models/Speaker.ts
import { SocialNetwork } from './SocialNetwork';
import { MyEvent } from './MyEvent';

export interface Speaker
{
    id: number;
    name: string;
    curriculum: string;
    imageURL: string;
    telephone: string;
    email: string;
    socialNetworks: SocialNetwork[];
    EventSpeakers: MyEvent[];
}
